libs/ballot-interpreter-vx/src/hmpb/find_targets.ts

```diff
@@ -26,7 +31,13 @@ export function* findTargets(
   const minAspectRatio = aspectRatio - aspectRatioTolerance;
   const maxAspectRatio = aspectRatio + aspectRatioTolerance;
 
-  const x = bounds.x + Math.round(inset + expectedWidth / 2);
+  const x =
+    bounds.x +
+    Math.round(
+      targetMarkPosition === BallotTargetMarkPosition.Left
```

Comment:
is there any chance that this causes an explicit `null` to default `BallotTargetMarkPosition.Right`, and thus should this conditional be checking that it equals `Right` and otherwise defaulting to `Left`?

I see the default parameter value, so maybe not, but I liked how your other conditionals were explicitly checking for `Right` and otherwise doing the `Left` thing.

Reply:
I agree that we should switch the conditional to check for `Right` as an extra measure of safety and for consistency. That said, passing `null` here would cause type checking to fail because an optional field allows `undefined` but not `null`. Here's a simplified example ([playground](https://www.typescriptlang.org/play?#code/KYOwrgtgBACg9gZwJYBclxFA3gKCvqAGWADMUoBeKAcgBtSVqAaPAgJSQHMALcq6gE5dezHAF8cOEmBABjNBigATOAEEAKtyQhOACixQADolTpMVeMgUgAdMTJQxlbGICU2VvlkYEcejdo4PQNjKzNHVwBucUkVDS0dXSicOM1tYKMTawAuKHBaWgjo1ISM0NMMXMsK2w4ecjdIoA)):

```ts

enum Position {
    Left = 'left',
    Right = 'right',
}

function doAThing({ position = Position.Left } = {}) {
    console.log({ position });
}

doAThing();
doAThing({ position: null });
doAThing({ position: Position.Right });
```

You get this error on the `null` line:

```
Type 'null' is not assignable to type 'Position | undefined'.
```